model/src/main/java/org/projectnessie/api/ContentApi.java

```diff
@@ -39,7 +39,8 @@
    *
    * <p>If the table-metadata is tracked globally (Iceberg), Nessie returns a {@link Content}
    * object, that contains the most up-to-date part for the globally tracked part (Iceberg:
-   * table-metadata) plus the per-Nessie-reference/hash specific part (Iceberg: snapshot-ID).
+   * table-metadata) plus the per-Nessie-reference/hash specific part (Iceberg: snapshot-ID,
+   * schema-ID, partition-spec-ID, default-sort-order-ID).
```

Comment:
nit: In the descriptions, If we are naming this variable(`default-sort-order-ID`) based on iceberg spec, then other two variable has to be `default-spec-id`, `current-schema-id`

or may be instead of mentioning each field, we just link it to `IcebergRefState`

Reply:
nit: i thikn `ID` shjould be `id`